backends/qualcomm/partition/qnn_partitioner.py

```diff
@@ -41,25 +39,20 @@ def __init__(self, graph_module: torch.fx.GraphModule, compiler_specs):
         self.qnn_manager.Init()
 
     def is_node_supported(self, _, node: torch.fx.Node) -> bool:
-        if node.op != "call_function":
+        if node.op != "call_function" or node.target in not_supported_operator:
             return False
 
-        if node.target in not_supported_operator:
-            return False
-
-        op_wrapper = self.node_visitors[node.target.__name__].define_node(
-            node, self.nodes_to_wrappers
-        )
         if node.target in allow_list_operator:
             return True
 
-        if op_wrapper is None:
-            print(f"[QNN Partitioner Op Support]: {node.target.__name__} | False")
-            return False
-
-        supported = self.qnn_manager.IsNodeSupportedByBackend(
-            [op_wrapper.GetOpWrapper()]
+        supported = False
+        op_wrapper = self.node_visitors[node.target.__name__].define_node(
+            node, self.nodes_to_wrappers
```

Comment:
nit: it feels nodes_to_wrapers should really be owned by node_visitor? And if it should be owned by partitioner, I would imagine then all the caching logic should be inside partitioner?

Reply:
Thank you for the idea. Since `node_visitor` is a dictionary, if `nodes_to_wrappers` was owned by each instance, we might need to traverse all for them to get the cached tensors.